packages/SwingSet/test/test-vat-env.js

```diff
@@ -2,7 +2,7 @@
 import '@agoric/swingset-vat/tools/prepare-test-env';
 import { Far } from '@agoric/marshal';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import test from 'ava';
+import { test } from '../tools/prepare-test-env';
```

Comment:
This looks like it needs to move to the top of the file, and replace the existing import of `prepare-test-env`. Although... if a secondary purpose of this test is to assert that the advice we're giving developers ("add `import '@agoric/swingset-vat/tools/prepare-test-env'` to the first line of your test programs") works, then it's important that we use that fully-qualified `import` and *not* a relative `../tools/...` one. (Consider a bug in which our directory structure got rearranged and the fully-qualified pathname no longer worked, but the relative one did.. or what if the package gets renamed entirely, or something).

Reply:
No secondary purpose. Just a simple mistake. Fixing.